src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs

```diff
@@ -27,4 +28,11 @@ public static void Method_CreateUserNameOverTransportBindingElement()
         Assert.True(element != null, "SecurityBindingElement added to binding elements should not be null.");
         Assert.Equal(element, securityBindingElement);
     }
+
+    [WcfFact]
+    public static void Property_DefaultAlgorithmSuite()
+    {
+        TransportSecurityBindingElement securityBindingElement = new TransportSecurityBindingElement();
+        Assert.Equal(securityBindingElement.DefaultAlgorithmSuite, SecurityAlgorithmSuite.Basic256);
+    }
```

Comment:
Did we add Basic256 to the contract?

Reply:
No, but we did just add SecurityBindingElement.DefaultAlgorithmSuite with #3909 so it was convenient to add a quick of the default value which is Basic256.